KILTprotocol/mashnet-node
# feat: add kilt-rococo spec

## fixes KILTProtocol/ticket#1748
* Adds KILT's rococo spec `RILT` by duplicating code from `WILT` and replacing the keys and DNS entries (discussed with @ggera)
* [M](https://kilt-cast.monday.com/boards/1792114080/pulses/2181459585)

## How to test:
I tested with a slighlty modified dev chain spec in out local-parachain-setup repository, see [this PR](https://github.com/KILTprotocol/local-parachain-setup/pull/13).

## Checklist:

  - [x] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
